### ISSUE TITLE
Error with `vagrant up` command

### ISSUE THREAD

**Issue by brianfeister** [santiagogil/Lost#1]
Hey @santiagogil - I'm getting the following error when I do `vagrant up`.  Have you seen this before?  I'm at commit #b050109dc98d0257545bb584e8c37be5c05744d6.

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

mount -t vboxsf -o uid=`id -u vagrant`,gid=`id -g vagrant` v-csc-1 /tmp/vagrant-chef-1/chef-solo-1/cookbooks
```

**Comment by santiagogil** [comment]
@brianfeister : Not yet. But let me see what can I find.

**Comment by brianfeister** [comment]
Found this, not sure if it's related: https://github.com/monigusto/vagrant-monigusto/issues/2

**Comment by santiagogil** [comment]
@brianfeister I found this http://vagrantup.com/v1/docs/troubleshooting.html
There was a kernel update lately. Maybe that's the problem. I can't reproduce the bug in my machine.

**Comment by brianfeister** [comment]
Unfortunately the troubleshooting advice there, suggests the kernel could be an issue if `lsmod | grep vboxsf` returns empty on the guest box, but for me it returns:

```
vboxsf                 42387  2 
vboxguest             198166  2 vboxsf
```

I managed to get past error by doing 

`mkdir cookbooks && touch cookbooks/.empty`
`git add -f cookbooks` 

Now, I'm getting `cookbook build-essential not found ...` so I'm guessing that's the wrong approach and that the `.empty` is causing the build process to skip over the import of the cookbooks...

**Comment by santiagogil** [comment]
@brianfeister : My mistake... The repo does'nt have the cookbooks or the gem librarian. Librarian it's the responsible for managing cookbooks and its dependencies. I have to include it or include the cookbooks in the repo. In the meantime I think that you can manage this by installing librarian gem and runing 

```
librarian-chef install
```

**Comment by brianfeister** [comment]
Thanks for the head's up!  I had a feeling I was missing something obvious
like that...

On Fri, Dec 28, 2012 at 6:06 AM, Santiago Gil notifications@github.comwrote:

> @brianfeister https://github.com/brianfeister : My mistake... The repo
> does'nt have the cookbooks or the gem librarian. Librarian it's the
> responsible for managing cookbooks and its dependencies. I have to include
> it or include the cookbooks in the repo. In the meantime I think that you
> can manage this by installing librarian gem and runing
> 
> librarian-chef install
> 
>  —
> Reply to this email directly or view it on GitHubhttps://github.com/santiagogil/Lost/issues/1#issuecomment-11729713.

## 

Brian Feister

Creative Director

## DreamStar Studios

(e) brian@dreamstarstudios.com | (p) 302.562.0308 | (w)
http://dreamstarstudios.com | Facebookhttp://facebook.com/dreamstarstudios

**Comment by brianfeister** [comment]
@santiagogil - I haven't even looked at this for a while, but you should try destroying your virtualbox and rebuilding.  I'm getting an error with the mysql recipe (this is after successfully installing librarian-chef and running `librarian-chef install`): 

```
package[percona-server-client-5.5] (mysql::client line 46) had an error: Chef::Exceptions::Package: No version specified, and no candidate version available for percona-server-client-5.5
```
